core/data/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/data/sessions/FakeSessionsApiClient.kt

```diff
@@ -84,7 +84,7 @@ fun SessionsAllResponse.Companion.fake(): SessionsAllResponse {
             isServiceSession = true,
             isPlenumSession = false,
             speakers = emptyList(),
-            roomId = 1,
+            roomId = 2,
```

Comment:
#907 changed the behavior how to sort rooms (by room name) so `arctic fox` is placed at the most left side now.
Because of this, I changed roomId `1` to `2`